### DISCUSSION TITLE
help please HA sensors not being created

### DISCUSSION THREAD

**Discussion by harphere** [ESPresense/ESPresense#668] [Support]
I've had one Android and one iOS device successfully acting as beacons for a long time until the last update to HA 2022.9.2. Now, both of my sensors are not appearing in HA but the device ids are appearing in MQTT.

For the life of me I cannot get these sensors created. I've tried renaming, restarting, deleting, recreating but no dice.

Can someone help me out or having the same problem?

`sensors.yaml`:
```
- platform: mqtt_room
  device_id: "iBeacon:<something>-100-1"
  name: 'H Presense'
  state_topic: 'espresense/devices/iBeacon:<something>-100-1'
  timeout: 10
  away_timeout: 120
- platform: mqtt_room
  device_id: "irk:<somethingelse>"
  name: 'N Presense'
  state_topic: 'espresense/devices/irk:<somethingelse>'
  timeout: 10
  away_timeout: 120
```
The timing of this is coincidentally aligned with the MQTT breaking change from HA but since this an mqtt_room platform, it is not related but interesting that it occurred at the same time.

**Comment by AlfredJKwack** [comment]
Hi. Just chiming in that I have a similar issue on an older HA install (2022.8.6)

Devices were paired using the new ui on the base station, hence the perhaps unusual topic names.

`sensors.yaml`
```
- platform: mqtt_room
  name: "Maya iPhone BLE"
  device_id: "Maya’s iPhone"
  state_topic: "espresense/devices/Maya’s iPhone"
  timeout: 5
  away_timeout: 120
```

I've confirmed there is indeed traffic/updates on the MQTT topic above.

Environment:
- ESPresense v3.0.4
- Home Assistant 2022.8.6 
- MQTT v6.1.3

**Reply by harphere** [reply]
My configuration.yaml was incorrectly pointing to my sensors.yaml file after adjusting for changes due to the MQTT breaking changes. User error afterall.

**Reply by AlfredJKwack** [reply]
You mean the change in yaml format as outlined in this HA doc? :  [home-assistant.io/source/_integrations/sensor.mqtt.markdown ](https://github.com/home-assistant/home-assistant.io/blob/31f5521ca72df404b3fa29a2d7f98109b141bb51/source/_integrations/sensor.mqtt.markdown)

Old style:
```
sensor:
  - platform: mqtt
    state_topic: "home/bedroom/temperature"
```

New style:
```
# Example configuration.yaml entry
mqtt:
  sensor:
    - state_topic: "home/bedroom/temperature"
```

**Reply by harphere** [reply]
Yes. It's all good now after fixing my !include statement
